crates/bevy_render/src/lib.rs

```diff
@@ -1,8 +1,5 @@
-// FIXME(15321): solve CI failures, then replace with `#![expect()]`.
-#![allow(missing_docs, reason = "Not all docs are written yet, see #3492.")]
-#![allow(unsafe_code)]
-// `rustdoc_internals` is needed for `#[doc(fake_variadics)]`
-#![allow(internal_features)]
```

Comment:
Do the same thing here :)

```suggestion
#![cfg_attr(any(docsrs, docsrs_dep), expect(internal_features))]
```